compiler/tests/fail/register_allocation/arm-m4/no_mmx.jazz

```diff
@@ -0,0 +1,10 @@
+export fn main () -> reg u32 {
+  reg u32 res;
+  #mmx reg u32 tmp;
```

Comment:
I disagree that this test should fail. The annotation could be ignored, with a warning.

Reply:
Hi Vincent, I am relatively agree with you. If a mmx annotation does not make sens it is certainly better to fail/ignore it in pretyping instead. Question: why do you think it should be a warning (portability ?)

Reply:
No, it’s the opposite of portability. For portability, you would like any `mmx` annotation to be silently ignored on platforms where it does not make sense. So maybe no warning would be better.